pkg/service/rpc/egress.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
-// 	protoc        v3.21.6
+// 	protoc-gen-go v1.28.1
```

Comment:
side note: it'd be great to use CI to generate these so we don't have these version number diffs.

Reply:
added but this is failing due to a permission error - https://github.com/livekit/livekit/actions/runs/4070633980/jobs/7011671528